conf/application.conf

```diff
@@ -140,4 +140,4 @@ calculator.version = 2.0
 variable.journey.enabled = true
 topup.journey.enabled = true
 confirmationWithDetailedBreakdowns.enabled = false
-fastTrackJourney.enabled = false
+fastTrackJourney.enabled = true
```

Comment:
is set to "true" intentional?

Reply:
yes, by default we want flags to be true and thats what QA wants